bigquery/test/tables.test.js

```diff
@@ -380,5 +449,65 @@ describe('bigquery:tables', function () {
         gzip: false
       }]);
     });
+
+    describe('main:insert', function () {
```

Comment:
Change `main:insert` to just `insert`. This `describe` clause is already nested under the `main` `describe` clause. Having `main:insert` is like calling this test `main main:insert`